39410: Update gitops workflows.md (#339)
replicatedhq/replicated-docs: docs/enterprise/gitops-workflow.md

          <td>Enter the branch name or leave the field blank to use the default branch.</td>
        </tr>
        <tr>
          <td>Path</td>
          <td>Enter the folder name in the repository where the application deployment file will be stored. If you leave this field blank, the Replicated app manager creates a folder for you. However, the best practice is to manually create a folder in the repository labeled with the application name and dedicated for the deployment file only.</td>
          </tr>
      </table>

    1. Select the action for the admin console to take when there is an update available. By default, the admin console create automatic commits to the branch.

    1. For **What content will it contain?**, select what type of asset to deliver to the Git repository. Using the **Rendered YAML** option results in a single, rendered YAML file being committed to the repository.

      ![GitOps Action](/images/gitops-action.png)

    1. Click **Finish setup** if this is a first time setup. For subsequent edits, click **Update settings**.

      The Gitops subtab opens and displays a public deployment key. The private key is stored securely in the admin console.

> 1. Click **Copy key** to copy the key. Click **repo settings page** and add the public deployment key in the repository settings page. Enable the write access permissions for this key, otherwise the commit fails when KOTS pushes the commit to the repository.

Reviewer: Lgtm, but this wording felt better to me.
```suggestion
1. Click **Copy key** to copy the key. Click **repo settings page** and add the public deployment key in the repository settings page. Enable the write access permissions for this key, otherwise KOTS will be unable to push the commit to the repository.
```

Author: Yep, I hear ya. Adapting your change, just using present tense instead, e.g. "cannot" instead of "will be unable to".
